.circleci/config.yml

```diff
@@ -78,14 +76,20 @@ jobs:
 
             # Exclude multi-process transport tests because
             # they appear to have some crashing problem...
-            EXCLUDE="Transport/TransportMultiProcTest*:$EXCLUDE"
+            # EXCLUDE="Transport/TransportMultiProcTest*:$EXCLUDE"
```

Comment:
debug vestige?

Reply:
Looks like I found and fixed the existing issues with TCP transport and TransportMultiProcTest, but I wanted to preserve this lines just in case for sometime...

Reply:
Can we add a comment here to explain why we leave this here?